node/node.go

```diff
@@ -692,11 +658,16 @@ func (node *Node) setupForBeaconValidator() {
 }
 
 func (node *Node) setupForNewNode() {
-	chanPeer := make(chan *p2p.Peer)
-	// Add network info serivce.
-	node.serviceManager.RegisterService(service_manager.NetworkInfo, networkinfo.New(chanPeer))
-	// Add staking service.
-	node.serviceManager.RegisterService(service_manager.Staking, staking.New(chanPeer))
+	chanPeer := make(chan p2p.Peer)
+	stakingPeer := make(chan p2p.Peer)
+
+	// Register staking service.
+	node.serviceManager.RegisterService(service_manager.Staking, staking.NewService(stakingPeer))
```

Comment:
can you change NewService to New? by golang documentation, as all packages under service consided as service.
See https://golang.org/doc/effective_go.html#package-names